### DISCUSSION TITLE
Welcome to online_ambulance_booking_service Discussions!

### DISCUSSION THREAD

**Discussion by MehaRima** [MehaRima/online_ambulance_booking_service#6] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Karanveer-web** [comment]
Does one has to install the exact versions of the softwares used?

**Reply by MehaRima** [reply]
Yes :)

**Reply by ndengeyingoma** [reply]
hello there this online ambulance booking service....on the admin login user name and password not working
